mern/client/src/Contexts/sessionProvider.js

```diff
@@ -1,16 +1,19 @@
-import React from "react";
+import React, { useEffect } from "react";
 
 export const SessionContext = React.createContext();
 
 export default function SessionProvider({ children }) {
   // nav state to show or hide session id/save/delete buttons
-  const [navState, setNavState] = React.useState(false);
+  const [sessionID, setSessionID] = React.useState();
+  useEffect(() => {
```

Comment:
This useEffect was used for debugging I'm assuming. Not sure if this should be pushed.